oldwinapi/windows.nim

```diff
@@ -8357,14 +8361,14 @@ type
     dbcp_size*: ULONG
     dbcp_devicetype*: ULONG
     dbcp_reserved*: ULONG
-    dbcp_name*: array[0..0, char]
+    dbcp_name*: UncheckedArray[char]
 
   PDEV_BROADCAST_PORT* = ptr DEV_BROADCAST_PORT
   TDEVBROADCASTPORT* = DEV_BROADCAST_PORT
   DEV_BROADCAST_USERDEFINED* {.final, pure.} = object
     dbud_dbh*: DEV_BROADCAST_HDR
-    dbud_szName*: array[0..0, char]
-    dbud_rgbUserDefined*: array[0..0, int8]
+    dbud_szName*: UncheckedArray[char]
+    dbud_rgbUserDefined*: UncheckedArray[int8]

```

Comment:
This field is impossible to use and according to https://msdn.microsoft.com/en-us/library/windows/desktop/aa363481(v=vs.85).aspx does not exist.

Reply:
Removed.